Create ui-c8y-1018-503-118-remove-angular-js-modal-for-adding-group-in-6617.md (#2404)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1018-503-118-remove-angular-js-modal-for-adding-group-in-6617.md

---
date: ""

> title: remove AngularJS modal for adding group in … (#6617)

Author: The Claude AI suggested to use the following title:
```suggestion
title: Remove modal for adding groups in device management
```

product_area: Application enablement & solutions
change_type:

>   - value: change-VSkj2iV9m

Reviewer: ```suggestion
  - value: change-2c7RdTdXo4
```

>     label: Fix

Reviewer: ```suggestion
    label: Improvement
```

component:
  - value: component-YbYJ3gLU_
    label: Web SDK
build_artifact:
  - value: tc-pjJiURv9Y
    label: ui-c8y
ticket: MTM-58710
version: 1018.503.118
---

> remove AngularJS modal for adding group in … (#6617)

Author: The Claude AI suggested to use the following body:
```suggestion
In the device management application, administrators previously used a modal dialog to add new device groups. To modernize the user interface and improve consistency, this modal dialog has been removed. Administrators can now add device groups directly in the device group list view without the need for a separate modal dialog. This change streamlines the workflow for managing device groups and brings the device management application in line with more modern user interface practices.
```
